packages/table/src/TableRow.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  variant as getVariantStyles,
+  VariantProps,
+  sx,
+  Theme,
+  css,
+  SerializedStyles,
+  styled,
+} from '@kodiak-ui/core'
+import { tableRoot } from './Table'
+
+export type TableRowProps = React.HTMLAttributes<HTMLTableRowElement> &
+  VariantProps
+
+export function base({ theme }: { theme: Theme }): SerializedStyles {
+  return css({
+    color: 'inherit',
+    display: 'table-row',
+    outline: 0,
+    verticalAlign: 'middle',
+  })(theme)
+}
+
+function variant({
+  variant: variantProp,
```

Comment:
Do we have the variants defined for these?  We might want a quick sample in the theme for the variants of the different sub components:

```
tables {
   tableRow: {}
}
```

Reply:
Yep, I can do that.